media/rendering.html

```diff
@@ -18,7 +18,7 @@
           <p>Beyond the declarative approach enabled by the <code>&lt;audio&gt;</code> element, the <a data-featureid="webaudio">Web Audio API</a> provides a full-fledged audio processing API, which includes support for low-latency playback of audio content.</p>
         </div>
 
-        <p data-feature="Rendering of captions"><a data-featureid="webvtt">WebVTT</a> is a file format for captions and subtitles. The specification is still a Working Draft, but the format is already widely supported among browsers, allowing to render text tracks through a <code>&lt;video&gt;</code> element. The <a data-featureid="ttml1">Timed Text Markup Language</a> (TTML) specification provides a richer language for describing timed text. It is used as an interchange format among authoring systems, in particular through profiles such as the <a data-featureid="ttml-imsc1">IMSC1 profile</a>.</p>
+        <p data-feature="Rendering of captions"><a data-featureid="webvtt">WebVTT</a> is a file format for captions and subtitles. The specification is still a Working Draft, but the format is already widely supported among browsers, allowing to render text tracks through a <code>&lt;video&gt;</code> element. The <a data-featureid="ttml1">Timed Text Markup Language</a> (TTML) specification provides a richer language for describing timed text. It is used as an interchange format among authoring systems, in particular through profiles such as the <a data-featureid="ttml-imsc1">IMSC1 profile</a>. Some browsers may not support IMSC1 natively, but note Web applications can still take advantage of IMSC1 in such cases through libraries such as the <a href="https://github.com/sandflow/imscJS">imscJS library</a> (a complete implementation of the IMSC1 profile in JavaScript).</p>
```

Comment:
I would say:

"The specification is still a Working Draft, but the format is already supported at various levels among browsers..."

and

"It is used both as an interchange format among authoring systems and for delivery of subtitles and captions worldwide, in particular through profiles such as the IMSC1 profile."

and

"such as the imscJS polyfill library, which is a complete implementation of the IMSC1 profile in JavaScript and renders IMSC1 documents to HTML5."